src/components/PopupWithConfirmation.js

```diff
@@ -0,0 +1,28 @@
+import Popup from "./Popup.js";
+
+export default class PopupWithConfirmation extends Popup {
+  constructor(popupSelector) {
+    super({ popupSelector });
+    this._yesDeleteButton = document.querySelector("#delete-card-button");
+    this._saveButton = this._popupElement.querySelector(".modal__save");
+  }
+
+setSubmitAction(action) {
+    this._handleSubmit = action;
+}
+// openPopup() {
+//     this._saveButton.textContent = "Yes";
+//     super.openPopup();
+//    }
+buttonTextYes() {
```

Comment:
Function names should start with a **verb** in infinitive (there is such a point in the checklist). It’s common in coding to do it so that functions could `speak` for themselves about what they do. **Please, correct it everywhere in the code**